application.jpaltime.e2e.yaml

```diff
@@ -0,0 +1,43 @@
+server:
```

Comment:
what do you think about putting all e2e-related files in either:
- `e2e` folder at the root of the repo
- `tests/e2e/files`

Reply:
Yep, it's a bit messy right now 👍